rfcs/text/i18n.md

```diff
@@ -6,40 +6,38 @@ Introduce first class support for internationalisation and localisation within `
 
 # Basic example
 
-The proposal is to introduce a new component `I18nProvider` that allows our consumers to provide a 
-[`react-i18next`](https://github.com/i18next/react-i18next) compatible<sup>[1] [2]</sup> `t` function.
+The proposal is to create a new `I18nProvider` component that allows carbon components to look 
+up translations by key. We will provide `en-GB` translations but we will accept pull requests to add translations for
+other common languages. We will only add complete locales to the `carbon-react` project. Consumers are able to extend
```

Comment:
Potentially this could add quite a bit of overhead going forwards. For example if we add the following:

* en-GB
* en-US
* en-CA
* fr-CA
* fr-FR
* de-DE
* es-ES
* es-US

Every time a new component gets added, or an existing one is changed - we will need to enforce this line about `complete locales` - requiring that the translations are updated for all those locales (which isn't an exhaustive list). These would also require review - which the core team might not be able to do without outside help

For me it would make sense for Carbon to default always to english, and allow consumers an easy way to provide translations for their own needs in their application (e.g. using I18next or I18n.js)

Reply:
Good point, I've added details about using `i18next` as a custom locale and removed references to locales other than `enGB`.